### ISSUE TITLE
ComboBox: aria-label is absent for list container

### ISSUE THREAD

**Issue by kangka29** [microsoft/fluentui#13705]
### Environment Information

- **Package version(s)**: office-ui-fabric-react@v7.117.4
- **Browser and OS versions**: all

### Sample app with Issue reproduced:
https://ref-styled-component.stackblitz.io

### Source code for Sample:
https://stackblitz.com/edit/ref-styled-component

###Prerequisites:
- Add [accessibility insights extension for web](https://accessibilityinsights.io/docs/en/web/overview).

#### Actual behavior:
When automated checks are executed for errors with accessibility insights for web extension, errors are reported as demonstrated below.

![image](https://user-images.githubusercontent.com/8017073/85121179-f4543880-b241-11ea-929e-7d45e35e4d13.png)

Please check failure 1.
![image](https://user-images.githubusercontent.com/8017073/85121257-16e65180-b242-11ea-99a3-08f8d390b644.png)

FYI, for failure 2, I earlier raised another [issue](https://github.com/microsoft/fluentui/issues/13525).

#### Expected behavior:

`aria-label` attribute should be available and no error should be reported in accessibility insights for web.

**Comment by naksenzh** [comment]
Hi Team, @behowell , please share the ETA for this bug

**Comment by Sanankh** [comment]
Hi Team, @behowell , please share the ETA for this bug

**Comment by JustSlone** [comment]
Hi, the issue here is that in the sample there is no label for the combobox nor is there an ariaLabel provided. 

```jsx
      <ComboBox
        defaultSelectedKey="B"
        ariaLabel="Descriptive Label" /*<--- this is the fix*/
        allowFreeform
        autoComplete="on"
        options={options}
      />
```
or 
```jsx
      <ComboBox
        defaultSelectedKey="B"
        label="Descriptive Label" /*<--- this is the fix*/
        allowFreeform
        autoComplete="on"
        options={options}
      />
```

Will resolve the error

**Comment by behowell** [comment]
@kangka29 @naksenzh @Sanankh - did JustSlone's suggestion fix the issue for you?

**Comment by melb47** [comment]
I am still seeing this accessibility error, even though the combobox control has the ariaLabel. The individual numbers in the combobox have appropriate ARIA labels, however, the container itself does not.
![image](https://user-images.githubusercontent.com/65355931/124958513-3cc6cb80-dfe8-11eb-8abc-b84b6ffd9253.png)

If I use label instead, it passes accessibility, however, it adds a visible label to the page which is unwanted.

**Comment by behowell** [comment]
It looks like the bug is that when you set `ariaLabel` on the ComboBox, it doesn't set the aria-label of the dropdown list element (with `role="listbox"`), is that correct? The expectation being that `<ComboBox ariaLabel="Example" />` would set the `aria-label` on both ComboBox's anchor element, _and_ dropdown list element. I'll submit a PR with the fix for that.

**Comment by melb47** [comment]
@behowell That sounds correct. In the Accessibility Insights, the `aria-labelledby="ComboBox214-label"` is referencing something that doesn't exist in the code, because there is no `label`, just `ariaLabel`: 
`<ComboBox
                styles={{
                    root: {
                        maxWidth: "55px",
                    },
                }}
                ariaLabel="Page"
                options={dropdownOptions}
                selectedKey={selectedPageIndex}
                onChange={(_, option) => onDropdownChange(option.index)}
                disabled={!totalPages}
            />`
